index.html

```diff
@@ -305,6 +305,50 @@ <h5 class="about-me-heading">About Me</h5>
         </ul>
       </div>
     </section>
-    
+    <footer class="footer">
+      <h2 class="footer-text">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h2>
+      <form action="https://formspree.io/f/xpzbopvv" method="POST">
+        <ul class="contact-section">
+          <li class="form-items">
+            <input
+              type="text"
+              name="name"
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) design as much as possible. Please remove the borders from all fields and make sure it matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/165702259-4b720889-bfd0-4c8e-a165-c0b36cafdf58.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/165702232-800b9a52-004c-4071-b123-685a01410967.png)